### ISSUE TITLE
Release swagger.json different to local one

### ISSUE THREAD

**Issue by hmajid2301** [netlify/open-api#231]
I'm using `netlify` version `4.1.4` (js-client), which in turn uses `open-api` version `0.13.2`. However
the imported `swagger.json` in the js-client is missing some endpoints. For example it doesn't have this endpoint `/dns_zones/{zone_id}/dns_records/{dns_record_id}`. However if I git clone this project then, run `npm convert` the endpoint is there in the generated `swagger.json`. I was wondering how I can resolve this in the netlify js client project ? I need to use that specific endpoint and client doesn't think it's a valid operation id as it's not present in the `swagger.json` file. 

TL:DR; The `swagger.json` present in the netlify js-client is different to the `swagger.json` that this project generates and open-api.netlify.com/.

**Comment by keiko713** [comment]
Hi! that endpoint is recently merged to the master (https://github.com/netlify/open-api/pull/211) but we haven't made a release yet, therefore you're not seeing it in a js-client yet.
I'm planning to cut a release either today or Monday. Actually js-client's open-api is using not up-to-date version open-api (current one is `0.13.4`), I'll get that updated too with the new release.

I'll update here once we've done releasing a new js-client version :)

**Comment by hmajid2301** [comment]
Yeh i did a diff between 0.13.4 and 0.13.3. Should've checked with master as well. Thanks anways. For the mean time I will manually copy over the `swagger.json` file I need into my `node_modules`.

Thanks

**Comment by keiko713** [comment]
it's released with https://github.com/netlify/js-client/releases/tag/v4.1.5 , thanks to @ehmicky 🎉 
closing this, but feel free to reopen if you have any further questions
